wandb/data_types.py

```diff
@@ -273,7 +270,7 @@ def __init__(self, columns=["Input", "Output", "Expected"], data=None, rows=None
         self.data = list(rows or data or [])
         if dataframe is not None:
             assert util.is_pandas_data_frame(dataframe), 'dataframe argument expects a `Dataframe` object'
-            self.columns = dataframe.columns.to_list()
+            self.columns = dataframe.columns.tolist()
```

Comment:
In cli-og we had to just call `list(dataframe.columns)` as some earlier versions of pandas didn't have `tolist`: https://github.com/wandb/client/blob/master/wandb/data_types.py#L264